superset/data/birth_names.py

```diff
@@ -231,7 +231,7 @@ def load_birth_names():
             The source dataset came from
             <a href='https://github.com/hadley/babynames' target='_blank'>[here]</a>
         </p>
-        <img src='/static/assets/images/babytux.jpg'>
+        <img src="https://i.imgur.com/aLOIXcC.png">
```

Comment:
This creates broken images if behind a firewall

Reply:
Shouldn't we assume that user's browsers have access to the internet?

Reply:
Ours don't ;-) it's there to prevent data leakage. I don't think you want your financial details to be out on the internet.

Reply:
Can we move forward with this? Mapbox and other things won't work from behind a computer that doesn't have access to the internet...

Reply:
Why not make it part of assets? MapBox *does* work behind a firewall if you pay for it or have the patience to compile your own. That's how we do it. I think it to be bad practice to rely on anything "outside". 

What if someone replaced that image with porn? And you open it up at work? Or at school? Malware served via images wouldn't be the first time either